patient_portal/index.php

```diff
@@ -104,23 +102,26 @@
           //default to english if only return one language
           $hiddenLanguageField = "<input type='hidden' name='languageChoice' value='1' />\n";
         }
-      }
-      else {
+    } else {
         $hiddenLanguageField = "<input type='hidden' name='languageChoice' value='".htmlspecialchars($defaultLangID,ENT_QUOTES)."' />\n";
       }
     }
     
 ?>
+<!DOCTYPE html>
 
 <html>
 <head>
     <title><?php echo xlt('Patient Portal Login'); ?></title>
 
-    <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>/jquery-min-1-11-3/index.js"></script>
-    <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>/jquery.gritter-1-7-4/js/jquery.gritter.min.js"></script>
-
-    <link rel="stylesheet" type="text/css" href="<?php echo $GLOBALS['standard_js_path']; ?>/jquery.gritter-1-7-4/css/jquery.gritter.css" />
+    <link rel="stylesheet" href="<?php echo $GLOBALS['standard_js_path']; ?>jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.min.css">
+    <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>jquery-min-1-11-3/index.js"></script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>jquery.gritter-1-7-4/js/jquery.gritter.min.js"></script>
```

Comment:
Gritter and all these have include_headers entries that can be used.

Reply:
Will apply the header code in subsequent releases